plugins/extensions/plugins/DeflectPlugin.cpp

```diff
@@ -102,12 +108,30 @@ bool DeflectPlugin::run(Engine& engine)
         _stream.reset();
     }
 
+    const bool observerOnly = engine.getParametersManager()
+                                  .getRenderingParameters()
+                                  .haveDeflectModule();
     if (deflectEnabled && !_stream)
-        _initializeDeflect();
+    {
+        if (_initializeDeflect(observerOnly))
+        {
+            const auto& windowSize = engine.getParametersManager()
+                                         .getApplicationParameters()
+                                         .getWindowSize();
+            deflect::SizeHints sizeHints;
+            sizeHints.preferredWidth = windowSize.x();
+            sizeHints.preferredHeight = windowSize.y();
+            sizeHints.minWidth = 64;
```

Comment:
Why 64? Magic value? or a parameter depending  on the engine? embree tile size? Deflect tile size?